pkg/kubelet/certificate/kubelet.go

```diff
@@ -63,6 +63,18 @@ func NewKubeletServerCertificateManager(kubeClient clientset.Interface, kubeCfg
 	)
 	legacyregistry.MustRegister(certificateExpiration)
 
+	certificateRotationAge := compbasemetrics.NewHistogram(
+		&compbasemetrics.HistogramOpts{
```

Comment:
The `server_expiration_seconds` above has a `StabilityLevel` field. Should the new `server_rotation_seconds` have that as well?

Reply:
:+1: